src/actions/partneraction.js

```diff
@@ -118,4 +118,19 @@ export const updatePartner = (data) => async (dispatch) => {
     } catch (err) {
       return Promise.reject(err);
     }
+};
+
+export const updatePendingRequestPartners = (data) => async (dispatch) => {
```

Comment:
no need to create new action it will duplicate, please use existing 'updatePartner' action